components/mdx/ComponentStyle.js

```diff
@@ -1,8 +1,14 @@
+import Link from "@components/Link";
+import Strong from "@components/mdx/Strong";
+import BlockQuote from "@components/mdx/BlockQuote";
```

Comment:
Blank link between import and export would be more readable and match other files